core/Updates/4.0.0-b1.php

```diff
@@ -121,6 +121,14 @@ public function doUpdate(Updater $updater)
             // switch to default provider if GeoIp Legacy was still in use
             LocationProvider::setCurrentProvider(LocationProvider\DefaultProvider::ID);
         }
+
+        // @todo migrate that to a config migration. See utf8mb4 branch
```

Comment:
as per comment. I've already built a config migration for the utf8mb4 branch. We should change that once that's merged. See https://github.com/matomo-org/matomo/pull/15618/commits/8907203134e0caaec482fe05a9b3f3cd4e20d386